src/components/menu/menuItem.js

```diff
@@ -0,0 +1,29 @@
+function menuItem({ name, ingredient, price }) {
+  const $menuItem = document.createElement('div');
+  $menuItem.setAttribute('class', 'd-flex flex-row justify-content-between px-5');
+
+  const $itemNamenAndIngredient = document.createElement('div');
+  // $itemNamenAndIngredient.setAttribute("class", "d-flex flex-column");
```

Comment:
- [ ] Comments: It’s better to delete commented piece of codes. If you need to review a previous version of your code, use git history

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png)

Reply:
Done.